docs/unity/quickstart.md

```diff
@@ -0,0 +1,319 @@
+
+## Creating tokens
+
+To create a token type, call the contract's `mint` entrypoint and pass these parameters:
+
+- A callback function to run when the token is created
+- The metadata for the token, which includes a name and description, URIs to preview media or thumbnails, and how many decimal places the token can be broken into
+- The destination account that owns the new tokens, which can be a user account, this smart contract, or any other smart contract
+- The number of tokens to create
+
+For example, this code creates a token type with a quantity of 100:
+
+```csharp
+var initialOwner = TezosManager
+    .Instance
+    .Wallet
+    .GetActiveAddress();
+
+const string imageAddress = "ipfs://QmX4t8ikQgjvLdqTtL51v6iVun9tNE7y7Txiw4piGQVNgK";
+
+var tokenMetadata = new TokenMetadata
+{
+    Name = "My token",
+    Description = "Description for my token",
+    Symbol = "MYTOKEN",
+    Decimals = "0",
+    DisplayUri = imageAddress,
+    ArtifactUri = imageAddress,
+    ThumbnailUri = imageAddress
+};
+
+TezosManager
+    .Instance
+    .Tezos
+    .TokenContract
+    .Mint(
+        completedCallback: OnTokenMinted,
+        tokenMetadata: tokenMetadata,
+        destination: initialOwner,
+        amount: 100);
+
+private void OnTokenMinted(TokenBalance tokenBalance)
+{
+    Debug.Log($"Successfully minted token with Token ID {tokenBalance.TokenId}");
+}
+```
+
+For a complete example of creating tokens, see the file `TezosSDK/Examples/Contract/Scripts/MintToken.cs` and the Contract example scene.
+
+## Transferring tokens
+
+To transfer tokens, call the contract's `Transfer` entrypoint and pass these parameters:
+
+- A callback function to run when the transfer is complete
+- The account to transfer the tokens to
+- The ID of the token
+- The amount of tokens to transfer
+
+This example transfers 12 tokens with the ID 5 to the account in the variable `destinationAccountAddress`.
+
+```csharp
+public void HandleTransfer()
+{
+    TezosManager
+        .Instance
+        .Tezos
+        .TokenContract
+        .Transfer(
+            completedCallback: TransferCompleted,
+            destination: destinationAccountAddress,
+            tokenId: 5,
+            amount: 12);
+}
+
+private void TransferCompleted(string txHash)
+{
+    Logger.LogDebug($"Transfer complete with transaction hash {txHash}");
+}
+```
+
+For a complete example, see the Transfer example scene.
+
+## Getting token balances
+
+To get the tokens that the connected account owns, call the [`API.GetTokensForOwner()`](./reference/API#gettokensforowner) method in a coroutine.
+This example prints information about the tokens that the account owns to the log:
+
+```csharp
+using Beacon.Sdk.Beacon;
+using TezosSDK.Beacon;
+using TezosSDK.Tezos;
+using TezosSDK.Helpers;
+using TezosSDK.Tezos.API.Models.Tokens;
+using TezosSDK.Tezos.API.Models.Filters;
+using System.Linq;
+
+// ...
+
```

Comment:
ok, removing these imports because a C# or Unity dev should know where to find the objects. I put them in there mostly because I didn't know an easy way to look up the imports when I copied a code block.

Reply:
Okay nice 👍  Most IDEs 99% of the time find and add these for you anyway. And if not, as you said. This code block is already too long, nice to shorten it.